nimble/host/src/ble_gap.c

```diff
@@ -2694,6 +2694,19 @@ ble_gap_ext_adv_params_tx(uint8_t instance,
     }
     if (params->legacy_pdu) {
```

Comment:
should we move this after include_tx_power to also cover anon an tx power?  (those are also invalid for legacy PDUs)

Reply:
Yes, absolutely.